rd-net/Test.RdFramework/Reflection/ProxyGeneratorAsyncCallsTest.cs

```diff
@@ -206,6 +210,24 @@ public async Task TestSyncCall()
       });
     }
 
+    [Test]
+    public async Task TestPrimitiveComposition()
+    {
+      await TestAsyncCalls(model =>
+      {
+        var vs = CFacade.Activator.Activate<IViewableSet<short>>();
+        vs.Add(123);
+        model.SyncMoments.Add(123, vs);
+        return Task.CompletedTask;
+      });
+
+/*#if NET35
+      // it seems like ExecuteSynchronously does not work in NET35
+      SpinWaitEx.SpinUntil(TimeSpan.FromSeconds(1), () => result != null);
+#endif*/
```

Comment:
So, does it or does not?

Reply:
It doesn't. It was copypasted from other test. 
Since 2019 we don't have tests in net35 5ef1848ad5f639d80295eac7f7db73e5d4c2a8e0. But it looks like it was removed accidentally. Should we reenable support for tests in net35?

Reply:
> Should we reenable support for tests in net35?

I think yes. Not in this particular PR, of course. Will you do that? Or ping me if it's too complex to do at once; I'll schedule it for later.

Reply:
It was too complex, I've made an issue
https://github.com/JetBrains/rd/issues/377